test/parallel/test-eventtarget.js

```diff
@@ -541,3 +541,16 @@ let asyncTest = Promise.resolve();
   et.addEventListener('foo', listener);
   et.dispatchEvent(new Event('foo'));
 }
+
+{
+  const target = new Event('test');
+  const constructorName = inspect(target, {
```

Comment:
Probably easier/better to just check `target.constructor.name` in this case?

Reply:
Also `EventTarget` could also benefit from this sort of test.

Reply:
> Also `EventTarget` could also benefit from this sort of test.

Nice catch! `EventTarget` test case has added.

Reply:
> Probably easier/better to just check `target.constructor.name` in this case?

This behavior is determined by `inspect`, so we need to use `inspect` in order to test the logic here.

https://github.com/nodejs/node/blob/c380ee6785b72fd2f24803de80bb412dfc32e8a9/lib/internal/event_target.js#L111-L114

Reply:
> This behavior is determined by inspect, so we need to use inspect in order to test the logic here.

The user can also just check `this.constructor.name` no? (the difference being that is also available (and required to be set correctly) by the spec)

Reply:
> The user can also just check `this.constructor.name` no? (the difference being that is also available (and required to be set correctly) by the spec)

You mean like this?

```js
const ev = new Event()
assert.strictEqual(
  ev.constructor.name,
  inspect(ev, { depth: -1 })
)
```

Reply:
No, literally:

```js
const ev = new Event()
assert.strictEqual(
  ev.constructor.name,
  "Event"
)
```

Which the DOM spec itself requires (the DOM spec isn't aware of `inspect` though that's fine to test too - just less interesting to me personally).

Reply:
I get it, but if we do this, we won't be able to improve the test coverage in this section. 
https://coverage.nodejs.org/coverage-0b6d3070a176d437/lib/internal/event_target.js.html#L111

This part of the code will only be executed with `inspect(ev, { depth: -1 })`

Reply:
sure, then let's keep it as is (and you can [if you feel like it] add a test for the constructor name directly since that's the part we should be careful not to break).

Reply:
> sure, then let's keep it as is (and you can [if you feel like it] add a test for the constructor name directly since that's the part we should be careful not to break).

Thanks for the careful guidance, I have learned a lot and new tests have been added!

Reply:
Thanks for the contribution - Node could definitely use more help in this space <3